src/gurt/telemetry.c

```diff
@@ -2366,15 +2362,14 @@ uint64_t *
 d_tm_get_shared_memory(int srv_idx)
 {
 	key_t	key;
-	int	shmid;
```

Comment:
Same question as above

Reply:
This one could stay local, but I was planning ahead for options if we wanted the data around for a consumer to issue the shmctl() at some point.  It's not strictly necessary to make this global at this time, but it doesn't hurt anything either.

Reply:
I'll revert it to a local shmid in this function.  We can always make it available in the future if needed.